drivers/scheduler/k8s/specs/postgres/pxd/px-storage-class.yaml

```diff
@@ -5,6 +5,15 @@ metadata:
   name: postgres-sc
 provisioner: kubernetes.io/portworx-volume
 parameters:
-  repl: "3"
+  {{ if .Repl }}
+  repl: "{{ .Repl }}"
+  {{ else }}
+  repl: "3"{{ end }}
   priority_io: "high"
-allowVolumeExpansion: true
+  {{ if .Fs }}
```

Comment:
are quotes not required here?
can we not populate Fs param if no value passed instead of defaulting to ext4?